schedule/yast/allmodules+allpatterns+registration/allmodules+allpatterns+registration.yaml

```diff
@@ -1,9 +1,18 @@
 ---
-name:           skip_registration
+name:           allmodules+allpatterns+registration
 description:    >
-  Like a standard scenario with explicit skipping of SCC registration
-  in case where we register by default, e.g. for SLE >= 15
-  See https://progress.opensuse.org/issues/25264 for details.
+  Full Medium installation that covers the following cases:
+     1. Additional modules enabled using SCC (Legacy, Development Tools, Web and
+        Scripting, Containers, Desktop Applications);
+     2. All patterns installed;
+     3. System registration is skipped during installation;
+     4. Installation is validated by successful boot and that YaST does not
+        report any issues;
+     5. Registration is performed on the installed system.
+vars:
+  SCC_REGISTER: ''
```

Comment:
I think it would be a good moment to document this variable in `variables.md` (at least with the use that we do from it because it was already there), wdyt?